src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -328,6 +328,11 @@ namespace HeatBalanceManager {
 
         GetBuildingData(state, ErrorsFound); // Read building data from input file
 
+        // need surface data to be set up before these add-on can be constructed
+        Material::GetMaterialAddOnInput(state, ErrorsFound); // Read variable thermal and solar absorptance addon data
+
+        Material::GetAddOnOverrideSurfaceList(state);
```

Comment:
This function should be part of `Surface`, not `Material`. Should not include any surface stuff in the Material module.

Reply:
Moved to HeatBalanceSurfaceManager

Reply:
well maybe move to DataSurfaces